core/build.gradle.kts

```diff
@@ -36,10 +37,18 @@ dependencies {
   testCompileOnly(libs.lombok)
 
   testImplementation(libs.awaitility)
-  testImplementation(libs.h2db)
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.junit.jupiter.params)
   testImplementation(libs.mockito.core)
 
   testRuntimeOnly(libs.junit.jupiter.engine)
 }
+
+tasks.test {
+  val testMode = project.properties["testMode"] as? String ?: "embedded"
```

Comment:
Do you need to replace MySQL backend with the H2 backend when running embedded jdbcBackend in IT (`com.datastrato.gravitino.integration.test.util.AbstractIT#startIntegrationTest`)?

Reply:
I don't think so, we will use the embedded JDBC backend to replace Key-value backend, and this won't affect the fact that we need to test JDBC(MySQL) in the CI.

Reply:
JDBC(MySQL) backend should be tested in deploy jdbcBackend?

Reply:
Not exactly. we can test JDBC(MySQL) in both embedded and deploy modes.  Gravitino start mode (embedded, deploy) has no relation to do with storage backend(embedded(h2), MySQL)